patroni/api.py

```diff
@@ -152,6 +152,11 @@ def do_GET(self, write_status_code_only=False):
                 status_code = replica_status_code
             elif path in ('/async', '/asynchronous') and not is_synchronous:
                 status_code = replica_status_code
+            elif path in ('/read-only-sync', '/read-only-synchronous'):
+                if primary_status_code == 200:
```

Comment:
I would suggest to include the standby_leader here:
```suggestion
                if 200 in (primary_status_code, standby_leader_status_code):
```